### ISSUE TITLE
Different results for QIC function after update

### ISSUE THREAD

**Issue by timktsang** [ekstroem/MESS#9]
When I use the QIC function in MESS 0.5.7, I can get the same results for
MESS::QIC() and gee pack::QIC()

However, when updated to MESS 0.5.9, I got the following error

Error in model.frame.default(formula = ambc ~ male + age + pm, data = data,  : 
  'data' must be a data.frame, environment, or list

May I know the the reason for this?

**Comment by ekstroem** [comment]
That should still work but please provide an example that reproduces the error. That will help me track it down.

**Comment by timktsang** [comment]
please see attached
[Archive.zip](https://github.com/ekstroem/MESS/files/9191192/Archive.zip)
.

**Comment by ekstroem** [comment]
It seems to be a scoping issue since your dataset is called `data` and there also is a global function called `data()`. If you call your dataset something else it should work for now while I try to figure out a check to prevent this from happening.

**Comment by timktsang** [comment]
yes. I tried, but another issue is that the QIC values were different from the one from geepack.

I also used the same code and using MESS 0.5.7, and the QIC from these two packages are the same.
[testing2.R.zip](https://github.com/ekstroem/MESS/files/9193713/testing2.R.zip)

please see attached code. Thank you.

**Comment by ekstroem** [comment]
That may be because you haven't updated geepack yet - an update was recently sent to CRAN for that too. The two functions are the same.

**Comment by timktsang** [comment]
I used the latest version (geepack 1.3.4). 

however, the output of geepack::QIC() and MESS::QIC() could be different, except for constr='independence'.

**Comment by ekstroem** [comment]
The latest version of geepack is 1.3.6. Get it from github or wait for it to be distributed via CRAN
